src/components/ScrollButton.tsx

```diff
@@ -0,0 +1,9 @@
+import Icon from './Icon';
+
+interface IArrow {
+    svg: string;
```

Comment:
is this the path to an svg? if so call it svgPath to denote that your expecting a string.

otherwise i would assume this receives an svg object